packages/core/src/components/resize-sensor/resizeSensor.tsx

```diff
@@ -79,12 +85,6 @@ export class ResizeSensor extends AbstractPureComponent<ResizeSensorProps> {
 
     public render(): React.ReactNode {
         const onlyChild = React.Children.only(this.props.children);
-
-        // if we're provided a ref to the child already, we don't need to attach one ourselves
-        if (this.props.targetRef !== undefined) {
-            return onlyChild;
-        }
```

Comment:
it turns out that this change broke usage of Popovers with children that are not DOM elements, such as `<Suggest>` which renders an `<InputGroup>` child. here's what happens:

1. ResizeSensor gets a ref callback as its `targetRef`
2. ResizeSensor is not able to peek into this ref callback (like it could with `.current` on ref _objects_) to get the DOM element, so it must create its own ref
3. ResizeSensor's own ref can only attach to its direct child, which may not be a native DOM element
4. ResizeSensor returns this non-DOM element value to `Popover#targetRef` at runtime, thereby setting `Popover#targetElement` to an invalid type (e.g. `InputGroup` inside a `Suggest`, instead of `HTMLInputElement`)
5. the page crashes because:
  i. `Popover` code which expects `this.targetElement` to be a DOM node fails (e.g. `element.closest()`)
  ii. popper.js fails because it doesn't have access to a valid DOM node